.github/pull_request_template.md

```diff
@@ -10,5 +10,12 @@ DEVPROD-NNNNN
 ### Testing
 <!-- add a description of how you tested it -->
 
+### Checklist
+- [ ] I have tested this code
+- [ ] I have updated the necessary documentation
+- [ ] I have added the necessary tests
+- [ ] I have updated the [analytics documentation](https://docs.google.com/spreadsheets/d/1s4_nq8ZiphXp5Uq_-9HT6GPqz-KOyaq6HuvmXYaSNzg/edit?usp=sharing) if necessary
```

Comment:
Maybe an unpopular opinion but I really don't like this on the app PR template 🫣 especially since the testing items feel redundant alongside our testing header. Would you be open to adding the analytics note as a comment? I'd also be happier if we just removed the testing questions